pkg/common/resource.go

```diff
@@ -99,12 +103,38 @@ func getResource(resourceList v1.ResourceList) *si.Resource {
 	return resources.Build()
 }
 
-func CreateUpdateRequestForTask(appID, taskID string, resource *si.Resource) si.UpdateRequest {
+func createTagsForTask(pod *v1.Pod, taskID string) map[string]string {
+	tags := map[string]string{
+		"namespace": pod.Namespace,
```

Comment:
I think we should add domain names for these tags.
For things from pod spec, I suggest we use the following keys:
- kubernetes.io/meta/podName
- kubernetes.io/meta/namespace
- kubernetes.io/label/some-label
- kubernetes.io/annotation/some-annotation

this is to give us the flexibility to add more tags in the future, we do have plan to introduce some yunikorn domain tags, and those will be like
- yunikorn.apache.org/group/some-tag
- yunikorn.apache.org/some-tag

this is using format `domain/group/key-name`. I do think we need to add `domain` and `group` as enum types in the scheduler-interface spec. So we can refer that in both core and the shim. Does this make sense to you?

Reply:
Namespace is an application level tag already. I don't think we should allow different namespaces for pods within an application and thus not allow specifying the namespace on the ask. It will be really confusing.

The labels should also be screened and not just all added to the tags. The tags for an application and an allocation ask were added to the SI for placement and allocation support not to just provide further details on the object.

Reply:
Created a [PR|https://github.com/apache/incubator-yunikorn-scheduler-interface/pull/14] for the constants in scheduler-interface. It seems that these constants are not added to the package, so I also added them to common/constants.go.

Reply:
Removed the namespace from the allocation tags.

Reply:
Removed bulk add of the labels.